eth/tracers/internal/tracers/assets.go

```diff
@@ -1,21 +1,21 @@
-// Code generated by go-bindata. DO NOT EDIT.
+// Code generated for package tracers by go-bindata DO NOT EDIT. (@generated)
 // sources:
-// 4byte_tracer.js (2.933kB)
-// bigram_tracer.js (1.712kB)
-// call_tracer.js (8.704kB)
-// evmdis_tracer.js (4.195kB)
-// noop_tracer.js (1.271kB)
-// opcount_tracer.js (1.372kB)
-// prestate_tracer.js (4.234kB)
-// trigram_tracer.js (1.788kB)
-// unigram_tracer.js (1.51kB)
-
+// .prestate_tracer.js.swp
```

Comment:
Is this some leftover tempfile that you left in the directory? Shouldn't be here -- there's no backing file in the repo